### DISCUSSION TITLE
How to browse notes quickly with keyboard shortcut

### DISCUSSION THREAD

**Discussion by kevinram** [dendronhq/dendron#3395] [General]
I can currently browse documents in the docs folder by switching to the explorer pane, using the down arrow, and then using the space bar to make the file display in the editor. 

Is there a command that I can assign to a shortcut, that will allow me to open the next file below the current one in the explorer, using just a single command?

**Comment by namjul** [comment]
Generally speaking the best way to browse dendron files would be to use [lookup](https://wiki.dendron.so/notes/a7c3a810-28c8-4b47-96a6-8156b1524af3/). To navigate between hierarchies dendron provides [sibling keybindings](https://wiki.dendron.so/notes/onsmbdh0r3f8k72fxbwz1ha/). 

> Is there a command that I can assign to a shortcut, that will allow me to open the next file below the current one in the explorer, using just a single command?

It appears that this once was possible with the command [`workbench.files.action.openNextWorkingFile`](https://vscode-docs.readthedocs.io/en/stable/customization/keybindings/#file-management) but was [changed](https://github.com/microsoft/vscode/issues/6605) to `workbench.action.nextEditor` which does not what you need. It navigates to the next open editor(tab) and so is not concerned with the explore pane anymore.
